CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 ### Changed
+
+- The `verify` command now manages Apalache server, if no already running
+  instance is detected (#1115)
+
```

Comment:
This is a feature add, not a breaking change, iiuc?

```suggestion
### Added

- The `verify` command now manages Apalache server, if no already running
  instance is detected (#1115)
  
### Changed
```

Reply:
Do we use these categories in a way that diverges from https://keepachangelog.com/?
They simple state

> `Changed` for changes in existing functionality.

Reply:
I will merge this PR, since Igor wanted to send install instructions for the summer school.
LMK if I should follow up with an updated changelog.

Reply:
Ah. I take "changes for existing functionality" to entail breaking changes. Otherwise, none of the categories entail breaking changes, and I also don't know how you can change existing functionality without that possibly breaking previous expectation about the API. That said, I can see how disagreement of the interpretation here is possible.

Apparently there has been a long-onging debate about this, lol: https://github.com/olivierlacan/keep-a-changelog/issues/41

Perhaps we should add a "Breaking changes" section?

I don't think my view is not widely enough shared to require any change to your entry.

Reply:
I see – yeah, it looks like the keepachangelog sections are a bit orthogonal to indicating breaking changes. Incidentally, also `Removed` may indicate a breaking change, so I guess in this taxonomy breaking changes are scattered across sections.

If we want to list breaking changes, we can either prefix individual items with a qualifier or break them into a separate section, as you suggested.